creator-node/src/services/stateMachineManager/stateReconciliation/StateReconciliationQueue.js

```diff
@@ -1,5 +1,273 @@
+const BullQueue = require('bull')
```

Comment:
maybe we should also have an addJob function here so that other queue has an abstraction layer instead of calling stateReconQueue.queue.add(...)

Reply:
good idea. I'll keep this conversation open so I remember to get to that when I implement the part that pipes monitoring job output into new reconciliation jobs